core/src/ephemeral_server/mod.rs

```diff
@@ -153,9 +153,7 @@ impl EphemeralServer {
         // TODO(cretz): Offer stdio suppression?
         let child = tokio::process::Command::new(config.exe_path)
             .args(config.args)
-            // We kill this on drop in case of errors or other issues. We still
-            // encourage users to manually shutdown.
-            .kill_on_drop(true)
+            .stdin(std::process::Stdio::null())
```

Comment:
This was also needed by @bergundy because on M1 environments the non-blocking stdin was preventing startup for whatever reason

Reply:
nit: I'd consider writing this comment in the code